src/app/pages/components/components.component.spec.ts

```diff
@@ -1,17 +1,17 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ComponentsComponent } from './components.component';
 
-import { PageOneComponent } from './page-one.component';
 
 describe('PageOneComponent', () => {
-  let component: PageOneComponent;
-  let fixture: ComponentFixture<PageOneComponent>;
+  let component: ComponentsComponent;
+  let fixture: ComponentFixture<ComponentsComponent>;
```

Comment:
Do we use these? They don't hurt I guess. 
Are they auto generated by something?

Reply:
Yeah they are autogenerated when you use ng generate to create a component

Reply:
spec.ts file is for testing.
 it is a default behaviour defined in angular.json file under schematics (that we can change also for simplicity).
 
 when using cli command:
 
"ng generate component"   you can  add "--skip-tests=true"  then it will not generate spec.ts file.

also when you create standalone components you can use below to avoid errors. 

"ng g c <component-name> --standalone"